### DISCUSSION TITLE
Welcome to ETC discord Discussions!

### DISCUSSION THREAD

**Discussion by realcodywburns** [ethereumclassic/discord#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TheCrowbill** [comment]
Hello,

I am Keith Hall Barnett, @TheCrowbill on Discord and most other social media platforms. I am a long-time (OG) member of the ETC Discord server and the Slack channel which preceded it. I began GPU-mining Ethereum in 2016 with my first rig coming online in June just weeks before the TheDAO contested hard fork. I was an investor in TheDAO, but getting back a few Ethereum at the cost of 'Code Is Law' and immutability held no appeal. I spoke out against all the proposed "fixes" made by the foundation. When the inevitability of a contentious hard fork became apparent, I sought out the advice of what few miner contacts I had at the time in search of a path that preserved the value of the technology as I saw it.  I found a mining pool that planned to continue the unforked chain, joined them before the split, and as such, have never mined a minute on ETH-the-forked.

Days after the fork, I was introduced to cryptocurrency trading by some new-found friends in the mining pool. My first exchange account was opened at Poloniex on what would later become the day Ethereum Classic was so named and trading began on that very exchange. Such coincidences placing me close, but not quite in, some historic ETC moments have made me feel somewhat like the Forrest Gump of ETC.

I attended Texas A&M University in the 1980s seeking a computer science degree which I did not complete. I have had a mostly entrepreneurial career in many sectors including technology, finance, construction, media, entertainment, energy, education, and agriculture.  At 58 years old, I am currently mostly retired. I live in rural Texas on a former cattle ranch.

Ethereum Classic, and general blockchain technology, has occupied a large portion of my world since its inception. As a former IRS enrolled agent familiar with the weaponization of taxation compliance enforcement by the US government, I immediately saw the potential freedom from government financial tyranny the technology could provide. Inspired, I took up coding again; something that was never my strong skill. Trained in RPG, COBOL and FORTRAN, acquiring a usable skillset regarding ETC has been an exercise in modernizing my skills which is an ongoing process. While no expert, I am becoming marginally functional in many of the currently relevant languages including JavaScript and Solidity. 

Currently, I am a member of the Ethereum Classic website and Twitter Together teams here on GitHub. I am exploring an opportunity to develop a smart contract suite for deployment on the ETC network. I am also one of the moderators of the ETC Discord server.
